src/renderer/components/item-object-list/item-list-layout.tsx

```diff
@@ -109,6 +113,22 @@ export class ItemListLayout extends React.Component<ItemListLayoutProps> {
     disposeOnUnmount(this, [
       reaction(() => toJS(this.userSettings), settings => storage.set(settings)),
     ]);
+
+    if (props.isResizable && props.tableId) {
```

Comment:
Like in your other table PR, I think this should be a `get canResize(): boolean`.